Vision/Vision.bs

```diff
@@ -54,19 +41,25 @@ Boilerplate: omit conformance
 		to represent the emergent consensus of most of our participants.
 	* Be timeless enough that it does not need frequent revision.
 
+# The Mission of W3C # {#mission}
+
+	The W3C leads the community 
+	in defining a World Wide Web that puts users first, 
+	by developing protocols and guidelines 
+	to empower an ethical, equitable, informed, and interconnected society.
+
 # Introduction # {#intro}
 
 	The World Wide Web was originally conceived
 	as a tool for sharing information.
-	It has evolved into a fundamental part of humanity,
+	It has evolved rapidly into a fundamental part of humanity,
+	sparking major social change by 
```

Comment:
Restructured this paragraph to simplify flow.

Reply:
+1 to this restructuring (which encompasses also the next diff chunk).